### ISSUE TITLE
Mog House 'aura' stength preventing removal of furnishings

### ISSUE THREAD

**Issue by Spiffly** [EdenServer/community#2608]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x] I have provided my client version in the details. (type /ver into your game window)

Ver = 30181205_0

I created this github report [https://github.com/EdenServer/community/issues/2494](url) previously but after further testing, i felt a new report was necessary as the original is only a by-product of the actual problem. It's a bit wordy so there's a tldr; at the bottom.

I believe Moghancement aura strength is preventing the removal of furnishings from the mog house. I have Moghancement: Gardening currently (1 x maple table (8), 10 x ceramic flowerpot (10) = aura strength 18). I wanted to replace this with Moghancement: Desynthesis. I placed 6 Tarutaru Stools (6 x 2 = aura strength 12) and then started to remove the ceramic flowerpots until Desythesis was the strongest effect. I was able to remove 6 flowerpots before i'm prevented from removing ANY further furnishings.

I was able to remove all of the ceramic pots and the maple table when i placed 9 Tarutaru stools. The game then prevents me from removing ANY of the stools. Basically i'm stuck with an aura strength 18 (and associated furnishings) in my mog house.

Tldr;

I believe if a furnishing is placed into the mog house, the game takes it's aura strength to be the new '0' and prevents it being removed as a negative aura strength isn't possible.

**Comment by Dribblet** [comment]
Make sure you have empty space in storage when removing furniture. You're unable to remove furniture if it results in your max storage space becoming a value lower than your current number of items stored. So 49/50 will prevent you from removing a white vase (4 storage, would result in 49/46 so disallowed), whereas removing 1 flowerpot is fine (1 storage, would result in 49/49 so allowed.)

**Comment by Spiffly** [comment]
Don't i feel stupid now! Thanks for clearing that up for me.
